### PR TITLE
chore: switch to using GitHub forms for issue creation

### PR DESCRIPTION
**Pull request by gkatsev**
This switches the bug and docs templates to use GitHub Forms via the yaml config. It also splits out Feature Request/Enhancement from the general Bug template.

You can view the form by viewing the file directly in github via the meatball menu.

Syntax for the types is here https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
Syntax for the top level options is here https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        - mux-audio-react
+        - mux-player
+        - mux-player-react
+        - mux-uploader
+        - mux-uploader-react
+        - playback-core
+        - common
+    validations:
+      required: true
+  - type: dropdown
```
Comment (cjpillsbury) [inline .github/ISSUE_TEMPLATE/feature_request.yml:39]: suggestion(non-blocking): Not sure if this is relevant for feature requests? consider removing. Or, if you can imagine a value add, minimally don't make required.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        - Chrome
+        - Safari
+        - Firefox
+        - Edge ("Edgeium")
+        - Chrome Android
+        - Safari on iOS
+        - Other (add details below)
+    validations:
+      required: true
+  - type: dropdown
```
Comment (cjpillsbury) [inline .github/ISSUE_TEMPLATE/feature_request.yml:55]: suggestion(non-blocking): Not sure if this is relevant for feature requests? consider removing. Or, if you can imagine a value add, minimally don't make required.

**Review by cjpillsbury — APPROVED** [review]
A few non-blocking suggestions but my primary feedback is: Gorgeous. Loves it. Can't wait.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        - mux-video
+        - mux-video-react
+        - mux-audio
+        - mux-audio-react
+        - mux-player
+        - mux-player-react
+        - mux-uploader
+        - mux-uploader-react
+        - playback-core
+        - common
```
[the remark this answers is not in the document]
Comment (gkatsev) [reply .github/ISSUE_TEMPLATE/feature_request.yml:36]: added `new element`

**Pull request merged**